LPS35HW.h

```diff
@@ -33,9 +33,19 @@ SOFTWARE.
 #define LPS35HW_DEFAULT_CTRL_REG1 0b00100010  // continuous@10Hz, no low pass filter, BDU on
 #define LPS35HW_DEFAULT_CTRL_REG2 0b00010000
 #define LPS35HW_ID 0b10110001  // same as LPS22HB, LPS33W, LPS33HW
-
+#define LPS35HW_DEFAULT_INTERRUPT 0b00000000
 class LPS35HW {
   public:
+    enum InterruptReg {
+        AUTORIFP    = 0b10000000,  // AUTORIFP: Enable AutoRifP function. Default value: 0
+        RESETARP    = 0b01000000,  // Reset AutoRifP function. Default value: 0
+        setAutoZero = 0b00100000,  // Enable Autozero. Default value: 0
+        RESET_AZ    = 0b00010000,  // Reset Autozero function. Default value: 0
+        DIFF_EN     = 0b00001000,  // Enable interrupt generation. Default value: 0
+        LIR         = 0b00000100,  // Latch interrupt request to the INT_SOURCE register. Default value: 0
+        PLE         = 0b00000010,  // Enable interrupt generation on differential pressure low event. Default value: 0 
+        PHE         = 0b00000001   // Enable interrupt generation on differential pressure high event. Default value: 0
```

Comment:
Please update these value to be camel case like the rest of the enums

Reply:
sorry @pilotak , I didn't wanted to pullrequest to you.
I close this pull and come back once it's finished and clean.
To much work to do until end of summer...
Thank you, simon